### ISSUE TITLE
Dashboard with libraries-bom

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/cloud-opensource-java#1263]
Right now the dashboard computes details for the various 250+ artifacts inividually. E.g. it looks at the latest google-cloud-vision and lists its linkage errors, local upper bounds failures, global upper bounds failures, and dependency convergence issues. 

We should also compute these values for each artifact when libraries-bom is used. That is, what is the state of each artifact if the version is not set directly but rather by importing libraries-bom. 

This shows how much improvement you get by using the BOM instead of importing an artifact directly. 

Step 1: Pick an artifact and figure out the numbers manually.

**Comment by suztomo** [comment]
I think we want diff of the output from

    cloud-oss-bom$ mvn help:effective-pom

since last release. A shell script with git, mvn, and diff will do the job. Dashboard is also nice.

**Comment by elharo** [comment]
No, I think that's something else. This is within the same release, and is the difference between the classpath with

```
<dependencies>
  <dependency>
    <groupId>com.google.cloud</groupId>
    <artifactId>google-cloud-storage</artifactId>
    <version>1.105.0</version>
  </dependency>
</dependencies>
```

and


```
<dependencyManagement>
 <dependencies>
  <dependency>
    <groupId>com.google.cloud</groupId>
    <artifactId>libraries-bom</artifactId>
    <version>4.2.0</version>
    <type>pom</type>
    <scope>import</scope>
   </dependency>
 </dependencies>
</dependencyManagement>

<dependency>
  <groupId>com.google.cloud</groupId>
  <artifactId>google-cloud-storage</artifactId>
</dependency>
```

The latter  might pull in more recent versions of transitive deps than the former.

**Comment by suztomo** [comment]
Thank you for clarification. 

I encountered the need of running Linkage Checker with managed dependencies when I tried pair-wise comparison: https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1022
